products.d/slowroll.yaml

```diff
@@ -0,0 +1,143 @@
+id: Slowroll
+name: Slowroll
+# ------------------------------------------------------------------------------
+# WARNING: There are currently no translations at this time!
+# ------------------------------------------------------------------------------
+description: 'A slightly slower rolling release of openSUSE designed to update
+  less often than Tumbleweed but more often than Leap without forcing users to
+  choose between "stable" and newer packages'
+icon: Slowroll.svg
+# Again, currently no translations at this time.
```

Comment:
```suggestion
# Do not manually change any translations! See README.md for more details.
```

Translations are automatically added through weblate, so it is unnecessary to do it by hand.